create_auction.php

```diff
@@ -1,13 +1,18 @@
 <?php include_once("header.php")?>
+<?php include_once("database.php"); ?>
+<?php include_once("database_functions.php"); ?>
 
 <?php
+session_start();
 /* (Uncomment this block to redirect people without selling privileges away from this page)
   // If user is not logged in or not a seller, they should not be able to
   // use this page.
   if (!isset($_SESSION['account_type']) || $_SESSION['account_type'] != 'seller') {

```

Comment:
Ideally, we should be doing something like this on every "log-in" gated page - we should be checking for the _$SESSION['logged_in'] value and redirect if the user is not logged in to the log in page.